src/runtime_src/core/common/drv/kds_core.c

```diff
@@ -1641,8 +1641,9 @@ int kds_cfg_update(struct kds_sched *kds)
 			if (!xcu)
 				continue;
 			if (!xcu->info.intr_enable) {
-				kds->cu_intr = false;
 				u32 cu_idx = xcu->info.cu_idx;
+
+				kds->cu_intr = false;
```

Comment:
Just curious, why do we need to change the order ?

Reply:
@chvamshi-xilinx  This is to cleanup the compile warning about "mixed declarations and code" which is not allow in C90 standard.

Thanks for merging the change.